itou/utils/templatetags/url_add_query.py

```diff
@@ -31,3 +31,10 @@ def url_add_query(url, **kwargs):
             querystring.pop(item)
     querystring.update(cleaned_kwargs)
     return urlunsplit(parsed._replace(query=querystring.urlencode("/")))
+
+
+@register.simple_tag
+def autologin_proconnect(url, user):
+    if user.is_authenticated and user.is_proconnect:
```

Comment:
Créer cette fonction juste pour ici, ça m'attriste beaucoup.

```suggestion
    if user.is_authenticated and user.identity_provider == IdentityProvider.PRO_CONNECT:
```

Reply:
ça ne mérite pas de tristesse … je l'ai tout de même enlevé ^^